src/services/jwt-service.ts

```diff
@@ -0,0 +1,19 @@
+import jwt from "jsonwebtoken";
+import jwtModel, { JwtModel } from "../models/jwt-model";
+
+export const jwtService = {
+  createToken: (id: string) => {
+    const secret = process.env.SECRET || "123456789";
```

Comment:
You are using this expression three different times in the project please create a utility function to get environment variables with their defaults